packages/theming/win32-theme/src/NativeModule/fallbackOfficeModule.ts

```diff
@@ -184,4 +160,10 @@ export const fallbackOfficeModule: OfficeThemingModule = {
   getPalette: fallbackGetPalette,
   typography: {},
   fluentTypography: {},
+  rampNames: {
+    App: ['FakeApp1', 'App2', 'App3', 'App4', 'App5', 'App6', 'App7', 'App8'],
```

Comment:
Right, so when I initially checked in the theming module I exposed just the values as arrays like you see in the `ramps` object, but later realized I wanted the value names as well, so there's an accompanying `rampNames` object.  I did it this way (instead of just a bunch `ramp` objects from the ThemingModule) to keep binary back compat in the off chance someone was using the ramp values directly. It was probably a little overly paranoid, but it didn't hurt.

I reassemble module.ramps and module.rampNames into proper objects in the createOfficeThemePartial code.

I probably could use the real values, but since these are constants and not a WebDebug-unsafe synchronous API call (like getPalette is) and you updated it so that the real module is used for all except the getPalette, the only time these get used is when the ThemingModule is fully mocked out like in the jest snapshots other platforms.

Including Fake is just an indicator that, hey, if you happen to see this, you're not getting real values.  Similar to how the TaskPanePalette is just the fallback WhiteColors but with `antiquewhite` shimmed in for Bkg.